src/components/NavDropdown/ExtendScale.html

```diff
@@ -0,0 +1,38 @@
+<span class="nav-dropdown-menu">
+  <ul>
+    <li>
+      <a href="/snaps/">
+        <span>
+          <svg width="800px" height="800px" viewBox="0 -0.5 21 21" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+            <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+              <g id="Dribbble-Light-Preview" transform="translate(-379.000000, -280.000000)" fill="currentColor">
+                <g id="icons" transform="translate(56.000000, 160.000000)">
+                  <path d="M341.9,137.261 C341.9,137.811 341.49785,138 340.92035,138 L331.47035,138 C330.89285,138 330.35,137.811 330.35,137.261 L330.35,135 L337.24535,135 C338.11475,135 338.75,134.589 338.75,133.761 L338.75,127 L340.92035,127 C341.49995,127 341.9,127.709 341.9,128.261 L341.9,137.261 Z M326.22035,133 C325.64285,133 325.1,132.811 325.1,132.261 L325.1,123.261 C325.1,122.709 325.6397,122 326.22035,122 L335.67035,122 C336.24995,122 336.65,122.709 336.65,123.261 L336.65,132.261 C336.65,132.811 336.24785,133 335.67035,133 L326.22035,133 Z M342.49535,125 L338.75,125 L338.75,121.761 C338.75,120.933 338.11475,120 337.24535,120 L324.64535,120 C323.7791,120 323,120.936 323,121.761 L323,133.761 C323,134.586 323.7791,135 324.64535,135 L328.25,135 L328.25,138.761 C328.25,139.586 329.0291,140 329.89535,140 L342.49535,140 C343.36475,140 344,139.589 344,138.761 L344,126.761 C344,125.933 343.36475,125 342.49535,125 L342.49535,125 Z" id="new_tab-[#1502]">
+                  </path>
+                </g>
+              </g>
+            </g>
+          </svg>
+        </span>
+        <div>
+          <h2>Snaps</h2>
+          <p>Create a custom Snap to extend the functionality of MetaMask.</p>
+        </div>
+      </a>
+    </li>
+    <li>
+      <a href="/services/">
+        <span>
+          <svg class="svg-inline--fa fa-iconInfuraNode fa-w-19 text-text-alternative group-hover:text-text-default" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" viewBox="0 0 16 14" role="img">
+            <path fill="currentColor" d="M0.46624 0.213623V2.27922L5.40032 1.77074H7.22144V5.1141L4.15488 6.05202L0 7.05394L0.61696 9.07346L4.5984 7.54322L7.22144 6.74162V12.2296H5.40032L0.46624 11.7208V13.7864H15.5338V11.7208L10.5997 12.2296H8.77856V6.74162L11.3885 7.53906L15.3834 9.07346L16 7.05394L11.8563 6.0549L8.77856 5.1141V1.77074H10.5997L15.5338 2.27922V0.213623H0.46624Z">
+            </path>
+          </svg>
+        </span>
+        <div>
+          <h2>Services</h2>
+          <p>Use services provided by Infura to optimize essential development tasks and scale your dapp or Snap.</p>
```

Comment:
"optimize essential development tasks" seems too vague. The main product Infura offers is APIs, right? And from the Infura website, there are 3 kinds of APIs: blockchain data, gas, and trace. The Infura site mentions developer tools, but that's a reference to all the MetaMask tooling. 

Also, it's not necessary to repeat the word "services." I think we can get a bit more specific by mentioning APIs and bring in some of the marketing language like "high performance." So: 

"Use high performance APIs provided by Infura to scale your dapp or Snap."